pkg/web_app/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
```

Comment:
Please write two lines at the top of this script that's says what it is used for... Like script to build client side dart to js..